src/unitxt/operators.py

```diff
@@ -376,6 +376,67 @@ def process_value(self, value: Any) -> Any:
         return new_value
 
 
+class AugmentSuffix(Augmentor):
+    """
+    Augments the input by appending to it a randonly selected (typically, whitespace) pattern.
+    Any existing trailig whitespace (if any), of the input is first discarded.
```

Comment:
Why you think we need this behavior?

Reply:
Thanks, @yoavkatz , 
Not sure.. thought to clean the plate before laying food on it. But you are right that I shouldnt impose it, and that the user can pre-clean through Apply. Removing now.